ietf/draft-ietf-calext-jscalendar-icalendar-09.txt

```diff
@@ -0,0 +1,6272 @@
+      to a dynamic rendition of the calendar object.
+
+   *  TBD - Choose STRUCTURED-DATA, if the schema property is set.
+
+   *  Choose ATTACH otherwise.
+
+   The href property converts to the iCalendar property value.  The
+   other Link properties convert as follows:
+
+
+
+
+
+
+
+
+
+Stepanek                   Expires 16 May 2025                 [Page 90]
+
+Internet-Draft                 JSCalendar                  November 2024
+
+
+     +=============+================+============+===================+
+     | Name        | Reference      | Parameter  | See               |
+     |             |                | (or other) |                   |
+     +=============+================+============+===================+
+     | cid         | [RFC8984],     | TBD        |                   |
+     |             | Section 1.4.11 |            |                   |
+     +-------------+----------------+------------+-------------------+
+     | contentType | [RFC8984],     | FMTTYPE    | Section 2.3.3     |
+     |             | Section 1.4.11 |            |                   |
+     +-------------+----------------+------------+-------------------+
+     | display     | [RFC8984],     | DISPLAY    | Section 2.3.23    |
+     |             | Section 1.4.11 |            |                   |
+     +-------------+----------------+------------+-------------------+
+     | rel         | [RFC8984],     | LINKREL    | Section 2.3.25,   |
+     |             | Section 1.4.11 |            | and remarks below |
+     +-------------+----------------+------------+-------------------+
+     | size        | [RFC8984],     | SIZE       | Section 2.3.3     |
+     |             | Section 1.4.11 |            |                   |
+     +-------------+----------------+------------+-------------------+
+     | title       | [RFC8984],     | TBD        |                   |
+     |             | Section 4.2.1  |            |                   |
+     +-------------+----------------+------------+-------------------+
+
+                  Table 34: Properties of the Link object
+
+   Remarks:
+
+   *  The LINKREL parameter SHOULD be omitted if the Link object
+      converts to an IMAGE property and the rel property value is
+      "icon".
+
+3.5.  Location
+
+   The Location object [RFC8984] (Section 4.2.5) converts to either a
+   LOCATION (Section 2.3.26), GEO (Section 2.3.22), or DTEND
+   (Section 2.3.22) property, or it converts to a VLOCATION component
+   (Section 2.2.4).  Which iCalendar element to choose is
+   implementation-specific.  As a guideline:
+
+   *  Choose LOCATION, if only the title property is set and the
+      iCalendar component not already contains a LOCATION property.  The
+      title property converts to the LOCATION property value.
+
+   *  Choose GEO, if only the coordinates property is set the iCalendar
+      component not already contains a GEO property.  The coordinates
```

Comment:
"is set the iCalendar component not already contains" -> "is set and the iCalendar component does not already contain" ?